code/modules/projectiles/ammo_datums.dm

```diff
@@ -1443,15 +1443,25 @@
 //unused and not working. need to refactor MD code. Unobtainable.
 //intended mechanic is to have xenos hit with it show up very frequently on any MDs around
 /datum/ammo/bullet/lever_action/tracker
-	name = "tracking lever-action bullet"
+	name = ".458 tracker round"
```

Comment:
If you want to change that you'll also need to change the ammunition and boxes etc to match, eg. `/obj/item/ammo_magazine/lever_action/tracker` which claims to be "45-70". Yes i know this is almost the same caliber but consistency is important

Reply:
This is actually a typo by me because the regular .458 rounds don't fit in the spotter rifle so I don't want to give users the idea that it will because they're called the same caliber.